### ISSUE TITLE
Example app referenced, but no example app available

### ISSUE THREAD

**Issue by greymouser** [adhearsion/mojo-auth#1]
The README references "An example application (a Rails app serving a WebRTC-based calling service backed by Kamailio and rfc5766-turn-server) is available to demonstrate usage."

The link points to this very repo: https://github.com/mojolingo/mojo-auth , but there is no app available here. The zip and tarball available on the GH Pages view just have the readme as well.

Could you please make the example app available so people like me can better wrap our collective heads around mojo-auth?

**Comment by benlangfeld** [comment]
Certainly. The example app is not actually written yet, but I'll throw it together this week. You'll see how simple it is :)
